### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for the "Star" images available  inside the extension links

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#49]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://marketplace.visualstudio.com/VSCode](https://marketplace.visualstudio.com/VSCode) in EDGE
2. Go to the "Star" images inside the "Extension Link" window available under "Featured", "Most Popular", "Trending" and "Recently Added" extension headers with mouse as keyboard is not accessible.
3. Observe the "Name" property in Inspect

**_Same issue is also observed with the link:_**
https://marketplace.visualstudio.com/items?itemName=donjayamanne.python
**Scenario-1:**
1. Go to the "Star Images" available in the "Python" blue colored window below the page header ( with mouse as keyboard is not accessible.)
2. Observe the "Name" property in Inspect

**Scenario-2:**
1. Go to the "Star Images" available in the "User Reviews" scetion of the page ( with mouse as keyboard is not accessible.)
2. Observe the "Name" property in Inspect

**Scenario-3:**
1. Click on "Write a Review" button available in the "User reviews" section of the page.
2. Navigate to the "Star Images" available with keyboard.
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Image" when navigated with CapsLock + -> keys in the keyboard

**Scenario-1:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Image" when navigated with CapsLock + -> keys in the keyboard

**Scenario-2:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Image" when navigated with CapsLock + -> keys in the keyboard

**Scenario-3:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Image" 

**Expected Result:**
Name property should indicate the purpose of images. So, Name property should be meaningful 

**User Impact:**
Visually impaired users  will not get to know the purpose of the image as the screen reader will not announce anything when the user arrives at the ratings image
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshots:**
![name property is empty for the star images](https://cloud.githubusercontent.com/assets/24825441/21718296/984b05a0-d43c-11e6-905c-918be8dbe499.png)
![sc-1 name property is empty](https://cloud.githubusercontent.com/assets/24825441/21759817/2de30aea-d66d-11e6-973a-fbe8dc7430ce.png)
![sc-2 name property is empty](https://cloud.githubusercontent.com/assets/24825441/21759816/2de0fb88-d66d-11e6-974b-d7c3bc815a88.png)
![sc-3 name property is empty](https://cloud.githubusercontent.com/assets/24825441/21759818/2de42682-d66d-11e6-91ba-d018249855c8.png)

**Comment by waderyan** [comment]
Marketplace

**Comment by waderyan** [comment]
Please send to Marketplace team.
